src/Helper.php

```diff
@@ -12,8 +13,9 @@ class Helper
     /**
      * Gets the id from an array, object or integer.
      *
-     * @param  mixed  $object
-     * @param  string  $type
+     * @param mixed  $object
+     * @param string $type
+     *
```

Comment:
If this PR is still relevant, please revert all unrelated changes like this reformatting.